sites-enabled/10-www.freecodecamp.org.conf

```
@@ -161,6 +161,20 @@ server {
     include snippets/common/proxy-params.conf;
   }
 
+  # reverse proxy news (italian) currently redirecting to english
+  location /italian/news {
+    if (-f /etc/nginx/maintenance/NEWS.txt) {
```

```suggestion
    if (-f /etc/nginx/maintenance/NEWS-ITA.txt) {
```

We use specific files to put up maintenance.

---

Yep, I kept this as English deliberately since it's redirecting to it.  My thinking was that you'd want this to be `NEWS-ITA.txt` once that exists, but for now you'd want its maintenance to be tied to the English.

---

If that's not right, please just go ahead and commit this.

---

You are actually correct, let's leave it as is. 

I would add a commented bit of code here so that it is not forgotten.

---

```suggestion
    # replace above with the next line, once there is a separate Italian /news
    # if (-f /etc/nginx/maintenance/NEWS.txt) {
```